app/views/hosts/show.html.erb

```diff
@@ -56,7 +56,7 @@
         <%= spinner(_('Loading VM information ...')) %>
         </div>
       <% end %>
-      <div id="nics" class="tab-pane" data-ajax-url='<%= nics_host_path(@host)%>' data-on-complete='onContentLoad'>
+      <div id="nics" class="tab-pane" data-ajax-url='<%= nics_host_path(@host)%>' data-on-complete='tfm.tools.activateTooltips'>
```

Comment:
There is no need to execute the entire function here, only tooltip activation is needed.